specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -475,9 +472,9 @@ app1.ear:
 
  META-INF/application.xml
 
- ejb1.jar Class-Path: util.jar
+ enterprisebean1.jar Class-Path: util.jar
```

Comment:
Since this was just an artifact of the application being described, there was no reason to change this from ejb1 and ejb2.  Just commenting, not going to hold up the approval.

Reply:
done